assets/quick-order-list.js

```diff
@@ -72,6 +72,22 @@ class QuickOrderList extends HTMLElement {
     this.quickOrderListId = 'quick-order-list'
     this.variantItemStatusElement = document.getElementById('shopping-cart-variant-item-status');
     const form = this.querySelector('form');
+    this.inputFiledHeight = this.querySelector('.variant-item__quantity-wrapper').offsetHeight;
```

Comment:
Is this a typo? Should it be "input**Field**Height"?

Reply:
It is 🤦‍♂️. Good catch. Thanks.